app/models/snpedia_paper.rb

```diff
@@ -9,4 +9,13 @@ class SnpediaPaper < ActiveRecord::Base
   def summary
     read_attribute(:summary).presence || "No summary provided."
   end
+
+  def matches_local_genotype?(other_local_genotype)
+    local_genotype == other_local_genotype.split('').sort.join
+  end
+
+  def local_genotype
+    url =~ /\((.*);(.*)\)$/
+    $1 + $2
+  end
```

Comment:
It's extracting the nucleotide pair from the SNPedia URL. The first method compares this with the `local_gentype` from the `UserSnp`, sorting the letters alphabetically, because they seem to be sorted in SNPedia as well.